frontend/src/components/EnvironmentConfiguration.jsx

```diff
@@ -576,6 +588,43 @@ export default function EnvironmentConfiguration (env) {
                             </Grid>
                         </Grid>
                         
+                        <Grid item xs={12}>
+                            <Grid container alignItems="center" direction="row">
+                                <Grid item xs={4}>
+                                    <InputLabel id="radius-label" sx={{ marginRight: 2, flexShrink: 0, color: '#F5F5DC', width: '200px' }}>
+                                        Enter radius (miles)
+                                    </InputLabel>
+                                </Grid>
+                                <Grid item xs={5}>
+                                    <TextField
+                                        sx={{
+                                            backgroundColor: '#F5F5DC',
+                                            '& .MuiOutlinedInput-root': {
+                                                '& .MuiInputBase-input': {
+                                                    padding: '6px 8px',
+                                                },
+                                            },
+                                        }}
+                                        id="Radius"
+                                        variant="outlined"
+                                        type="number"
+                                        inputProps={{ 
+                                            step: "0.1",
+                                            min: "0"
+                                        }}
+                                        onChange={handleOriginChange}
+                                        value={envConf.Origin.Radius === 0 || envConf.Origin.Radius === '' ? '' : envConf.Origin.Radius}
```

Comment:
@syedomair0 You can directly use `envConf.Origin.Radius` and remove ternary operator from here